doc/api/webcrypto.md

```diff
@@ -48,7 +48,7 @@ async function generateAesKey(length = 256) {
 }
 ```
 
-#### Elliptic curve key pairs
+#### ECDSA key pairs
```

Comment:
I would prefer if the whole acronym was mentioned here. IIRC that's "Elliptic Curve Digital Signature Algorithm" but I am not sure.

Reply:
We could maybe use `Elliptic Curve Digital Signature Algorithm (ECDSA)`, but it would make the heading very long and we also don't spell out slightly more commonly used acronyms (`AES`, `RSA`, `HMAC`, ...). Maybe we should add a sentence or two under each heading in a follow-up PR, explaining what algorithms the acronyms are referring to?

Reply:
Yeah or even just links to Wikipedia